.gitignore

```diff
@@ -39,6 +39,7 @@ scan_derived_data/
 generated_docs/
 Examples/**/Package.resolved
 Tests/**/Package.resolved
+!Tests/TestingApps/SimpleApp/SimpleApp.xcworkspace/xcshareddata/swiftpm/Package.resolved
```

Comment:
Hmm this might be slightly dangerous since, if we ever move this app somewhere else, we might forget to update this. How about updating the line above to be multiple entries for any other occurences of `Package.resolved`?

Reply:
Good point, good idea.

Reply:
Done.